test_site/settings.py

```diff
@@ -53,6 +53,7 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'method_override.middleware.MethodOverrideMiddleware',
```

Comment:
Para qué es esto?

Reply:
Lo necesito para enmascarar los métodos de DELETE y PUT en HTML, si lo quito me da error 405 method not allowed cuando borro o edito una silla